frontends/concrete-python/examples/levenshtein_distance/levenshtein_distance.py

```diff
@@ -269,7 +284,7 @@ def levenshtein_clear(x, y):
 
 
 @lru_cache
-def levenshtein_simulate(module, x, y):
+def levenshtein_simulate(module: fhe.module, x: str, y: str):
```

Comment:
This is not `fhe.module`, `fhe.module` is a decorator, I think we should use `fhe.Module` CC @aPere3

Reply:
I tried fhe.Module the first time (as you said) but I got:

```
python levenshtein_distance.py --distance Zam ma --alphabet StRiNg
Traceback (most recent call last):
  File "levenshtein_distance.py", line 287, in <module>
    def levenshtein_simulate(module: fhe.Module, x: str, y: str):
AttributeError: module 'concrete.fhe' has no attribute 'Module'
```

What should I do, @aPere3 @umut-sahin ?

Reply:
Postponed to later, https://github.com/zama-ai/concrete-internal/issues/811

Reply:
Yes, you should use `fhe.Module`.

Reply:
but it doesn't work @aPere3 , see https://github.com/zama-ai/concrete-internal/issues/811